### DISCUSSION TITLE
文件已由远程服务器锁定

### DISCUSSION THREAD

**Discussion by woodychanjm** [AlistGo/alist#5296] [General]
环境信息：群晖dsm7.2，使用docker-compose安装3.3版本的alist。
挂载夸克网盘后，使用群晖cloud sync进行双向备份测试。
一个大文件上传上传完成后，却提示上传失败并提示‘文件已由远程服务器锁定’，（此时在网盘上是可以正常观看该大文件的，应该是已经完成上传了的）

错误信息如图所示：
![image](https://github.com/alist-org/alist/assets/131585025/5843e77a-14cd-4b93-8e8e-0014d6d981df)

**Comment by w-kun** [comment]
我也遇到了，用的是115。开了加密就像上面那样提示上传失败，不开加密就会变成同一文件上传多次
![Snipaste_2023-10-06_19-53-44](https://github.com/alist-org/alist/assets/27620657/c88c0836-f99f-443a-90d2-5a88515356c1)

**Reply by echogroot01** [reply]
我用群晖cloud sync上传到onedrive也遇到这个报错，71个文件就1个文件不行，不知道是cloud sync的问题，还是alist webdev的问题

**Comment by xintonglaile** [comment]
一样一样的耶。用cloud sync上传就会这样子。

**Comment by naah69** [comment]
貌似文件比较大的时候，就有这个问题

**Comment by YYG501** [comment]
最后解决了吗
